test/jdk/java/awt/Dialog/FileDialogIconTest/FileDialogIconTest.java

```diff
@@ -156,7 +168,8 @@ public void actionPerformed(ActionEvent event) {
                     for (int i = 1; i <= 4; i++) {
                         String fileName = fileBase + "T" + i + ".gif";
                         image = Toolkit.getDefaultToolkit().getImage(fileName);
-                        PassFailJFrame.log("Loaded image " + fileName + ". setting to the list for dialog");
+                        PassFailJFrame.log("Loaded image " + "T" + i + ".gif."
+                                           + " Setting to the list for dialog");
```

Comment:
This piece of code looks the save as the one above.

Does it make to isolate the loading of images from setting them (to avoid code duplication)?